R/stat-bin.r

```diff
@@ -1,10 +1,10 @@
 #' @param binwidth The width of the bins. Can be specified as a numeric value
-#'   or as a function that calculates width from unscaled x. Here, "unscaled x" 
-#'   refers to the original x values in the data, before application of any 
+#'   or as a function that calculates width from unscaled x. Here, "unscaled x"
+#'   refers to the original x values in the data, before application of any
 #'   scale transformation. When specifying a function along with a grouping
-#'   structure, the function will be called once per group. 
+#'   structure, the function will be called once per group.
 #'   The default is to use `bins`
-#'   bins that cover the range of the data. You should always override
```

Comment:
I think this might have been intentional, i.e. "The default is to use `bins` [i.e. the `bins` parameter] number of bins". Maybe another way to word this would be "The default is to use the number of bins in `bins`, covering the range of the data"?